package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "wekan",
+  "version": "1.0.0",
+  "description": "The open-source Trello-like kanban",
+  "private": true,
```

Comment:
Marking a package as `private` will suppress multiple warnings of npm. It will also make sure you never accidentally `$ npm publish` this repository as an npm package.